awx/ui_next/src/screens/Inventory/shared/InventorySourceSubForms/SCMSubForm.jsx

```diff
@@ -122,20 +122,13 @@ const SCMSubForm = ({ autoPopulateProject, i18n }) => {
           onSelect={(event, value) => {
             setIsOpen(false);
             value = value.trim();
-            if (!value.endsWith('/')) {
-              value += '/';
-            }
             sourcePathHelpers.setValue(value);
           }}
           aria-label={i18n._(t`Select source path`)}
           placeholder={i18n._(t`Select source path`)}
           isCreatable
           onCreateOption={value => {
             value.trim();
-
-            if (!value.endsWith('/')) {
-              value += '/';
-            }
```

Comment:
This _may_ be addressing a different use case or bug that I am not aware of...

Reply:
@AlexSCorey Can these custom inventories ever be whole directories? :thinking: (Trying to make sure I don't miss something by removing this)

Reply:
I feel like removing this code is probably fine? Although I can't think of a use case for this off the top of my head, if someone _does_ need to append a `/` to a file name they can always do it manually:

![inv_slash](https://user-images.githubusercontent.com/9753817/112549038-8e2db080-8d93-11eb-9342-56e6eca09d91.gif)

cc @AlexSCorey - thoughts?